lms/djangoapps/certificates/apis/v0/views.py

```diff
@@ -7,14 +7,44 @@
 from edx_rest_framework_extensions import permissions
 from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
 from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
-from lms.djangoapps.certificates.api import get_certificate_for_user
+from lms.djangoapps.certificates.api import get_certificate_for_user, get_all_certificates
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 from openedx.core.lib.api.authentication import OAuth2AuthenticationAllowInactiveUser
 
 
 log = logging.getLogger(__name__)
 
+class CertificatesView(APIView):
```

Comment:
This is returning `NameError: name 'APIView' is not defined`. Please import that